docs/src/test/resources/application.conf

```diff
@@ -4,4 +4,4 @@ play.modules.disabled += play.api.db.DBModule
 play.modules.disabled += play.api.db.HikariCPModule
 play.modules.disabled += com.lightbend.lagom.javadsl.persistence.jdbc.JdbcPersistenceModule
 
-lagom.serialization.play-json.serializer-registry = com.lightbend.lagom.scaladsl.playjson.EmptySerializerRegistry
\ No newline at end of file
+lagom.serialization.play-json.serializer-registry = com.lightbend.lagom.scaladsl.playjson.EmptyJsonSerializerRegistry
```

Comment:
Good catch - I missed deleting it.  IntelliJ updated it automatically when I renamed the class.